src/map/clif.c

```diff
@@ -7104,8 +7106,10 @@ void clif_vendinglist(struct map_session_data* sd, int id, struct s_vending* ven
 
 #if PACKETVER < 20150226
 	const int item_length = 22;
-#else
+#elif PACKETVER < 20160921
 	const int item_length = 47;
+#else
+	const int item_length = 59;
```

Comment:
@Lemongrass3110  already added packet length 59 for PACKETVER condition 20160921 in the void clif_vendinglist..